homeassistant/components/myq/cover.py

```diff
@@ -106,23 +98,23 @@ async def async_close_cover(self, **kwargs):
         try:
             wait_task = await self._device.close(wait_for_state=False)
         except MyQError as err:
-            _LOGGER.error(
-                "Closing of cover %s failed with error: %s", self._device.name, str(err)
-            )
-
-            return
+            # Write current state which would be Open to HASS
+            self.async_write_ha_state()
```

Comment:
```suggestion
            # Write current state which would be Open to HASS
            self.async_write_ha_state()
```

If the action failed, the state should not have changed since the state hasn't been written yet.